OneWelcomeExampleApp/Interactors/RegisterUserInteractor.swift

```diff
@@ -45,8 +45,19 @@ extension RegisterUserInteractor: RegisterUserInteractorProtocol {
         return userClient.identityProviders
     }
 
-    func startUserRegistration(identityProvider: IdentityProvider? = nil) {
-        userClient.registerUserWith(identityProvider: identityProvider, scopes: ["read", "openid"], delegate: self)
+    func startUserRegistration(identityProvider: IdentityProvider?) {
+        guard let identityProvider else {
```

Comment:
Under the button we have `nil` that used to pick (later) default identity provider. With this guard it will never happen and we end up with error.
```
@IBAction func signUp(_: Any) {
        registerUserViewToPresenterProtocol.signUp(nil)
    }